include/cantera/thermo/ThermoPhase.h

```diff
@@ -975,6 +974,118 @@ class ThermoPhase : public Phase
      */
     virtual void setState_SV(doublereal s, doublereal v, doublereal tol = 1.e-4);
 
+    //! Set the specific entropy (J/kg/K) and temperature (K).
+    /*!
+     * This function fixes the internal state of the phase so that the specific
+     * entropy and temperature have the value of the input parameters.
+     * It must be implemented in a derived class.
```

Comment:
Hmm, I don’t know specifically what Ray finds misleading, but something along the lines of “Implementation must occur at the level of the derived class; absent this, the method throws a “NotImplemented” error” might be more detailed?


> On Jan 20, 2016, at 1:36 PM, Bryan W. Weber <notifications@github.com> wrote:
> 
> In include/cantera/thermo/ThermoPhase.h <https://github.com/Cantera/cantera/pull/315#discussion_r50314799>:
> 
> > @@ -975,6 +974,118 @@ class ThermoPhase : public Phase
> >       */
> >      virtual void setState_SV(doublereal s, doublereal v, doublereal tol = 1.e-4);
> >  
> > +    //! Set the specific entropy (J/kg/K) and temperature (K).
> > +    /*!
> > +     * This function fixes the internal state of the phase so that the specific
> > +     * entropy and temperature have the value of the input parameters.
> > +     * It must be implemented in a derived class.
> It is there to indicate that this method will throw a NotImplemented error if it is used without implementing it. That phrasing is present on a bunch of the similar functions. Is there an alternative phrasing you would prefer?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/Cantera/cantera/pull/315/files#r50314799>.
>